Overview.src.html

```diff
@@ -2306,8 +2329,8 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <h3>Main fetch</h3>
 
-<p>To perform a <dfn title=concept-main-fetch>main fetch</dfn> using <var>request</var>, optionally
-with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
+<p>To perform a <dfn title=concept-main-fetch data-export>main fetch</dfn> using <var>request</var>,
+optionally with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
```

Comment:
Ok. I poke at many of these (though non-normatively) from CSP to explain the call flow for various algorithms, but I can do those as one-offs if you don't want to export it generally.